frontend/src/employees/EmployeeListRow.js

```diff
@@ -0,0 +1,43 @@
+import React, {useState} from 'react'
+import { putUserInactive } from '../api/UserAPI';
+import EditButton from '../EditButton';
+import { Link } from 'react-router-dom';
+import Button from '../Button';
+
+export default function EmployeeListRow({user}) {
+
+    const [isActive, setIsActive] = useState(true)
+
+    function updater() {
+        putUserInactive(user.id)
+        setIsActive(false)
+    }
+    return (
+        <tr key={user.id}>
+        <th scope="row">{user.id}</th>
+        <td>{user.lastName} {user.firstName}</td>
+        <td>{user.email}</td>
+        {user.enabled && isActive
```

Comment:
Ha jól látom a 2 eset között csak a szöveg "Aktív" és "Inaktív" illetve a plusz Inaktiválá button a különbség. Szerintem szebb lenne ha a feltétel csak arra lenne rárakva, akkor nem kell duplikálni a td-t és linket, szerintem rövidebb is lenne ez a kód.

Reply:
Ahogy itt is van: #97  - frontend/src/employees/TableListOfEmployees.js